ci/build.sh

```diff
@@ -129,6 +129,16 @@ if [ -n "$GITHUB_TOKEN" ] && [ "$TRAVIS_EVENT_TYPE" == "pull_request" ]; then
   # Copy lines of prometheus-operator section and remove indention from values.yaml
   sed -n "$prometheus_start,${prometheus_end}p" deploy/helm/sumologic/values.yaml | sed 's/  //' >> deploy/helm/prometheus-overrides.yaml
 
+  echo "Copy prometheus.prometheusSpec.remoteWrite from 'prometheus-overrides.yaml' and inject into 'deploy/kubernetes/kube-prometheus-sumo-logic-mixin.libsonnet'"
+  prometheus_remote_write=`yq r deploy/helm/prometheus-overrides.yaml prometheus.prometheusSpec.remoteWrite -j | jq '.' | sed 's/^/    /'`
+  # Escaping so sed will work
+  prometheus_remote_write="${prometheus_remote_write//\\/\\\\}"
+  prometheus_remote_write="${prometheus_remote_write//\//\\/}"
+  prometheus_remote_write="${prometheus_remote_write//&/\\&}"
+  prometheus_remote_write="${prometheus_remote_write//$'\n'/\\n}"
+  echo "// This file is autogenerated" > deploy/kubernetes/kube-prometheus-sumo-logic-mixin.libsonnet
+  sed 's#\[\/\*REMOTE_WRITE\*\/\]#'$prometheus_remote_write'#' ci/jsonnet-mixin.tmpl | sed 's#"http://collection-sumologic.sumologic.svc.cluster.local:9888\/#$._config.sumologicCollectorSvc + "#g' | sed 's/+:     /+: /' | sed -r 's/"(\w*)":/\1:/g' > deploy/kubernetes/kube-prometheus-sumo-logic-mixin.libsonnet
+
```

Comment:
I think we  need to also add `$(git diff deploy/kubernetes/kube-prometheus-sumo-logic-mixin.libsonnet)` to the section commiting the generated files to git below (L152-L156)

Reply:
You're right, I'll add that.

Reply:
Fixed in 61ae61963550371f465824dcaef30ba48efaef54